### PR TITLE
T-13510 feat: implement all load balancers relays

### PR DESCRIPTION
**Pull request by pablocampogo**
Implement all load balancers relays endpoint for relay meter

### CODE REVIEW

**Review by commoddity — DISMISSED** [review]
Looks good ,thank you.

File: cmd/apiserver/main.go [hunk 1]
```diff
@@ -92,7 +92,7 @@ type backendProvider struct {
 func (b *backendProvider) UserApps(user string) ([]string, error) {
 	// TODO: make the timeout configurable
 	ctx, _ := context.WithDeadline(context.Background(), time.Now().Add(10*time.Second))
-	req, err := http.NewRequestWithContext(ctx, http.MethodGet, fmt.Sprintf("%s/user/%s/application", b.backendApiUrl, user), nil)
+	req, _ := http.NewRequestWithContext(ctx, http.MethodGet, fmt.Sprintf("%s/user/%s/application", b.backendApiUrl, user), nil)
```
Comment (adshmh) [inline cmd/apiserver/main.go]: Could we add a check on the error, instead of ignoring it?

File: cmd/apiserver/main.go [hunk 2]
```diff
@@ -125,7 +125,7 @@ func (b *backendProvider) UserApps(user string) ([]string, error) {
 func (b *backendProvider) LoadBalancer(endpoint string) (*repository.LoadBalancer, error) {
 	// TODO: make the timeout configurable
 	ctx, _ := context.WithDeadline(context.Background(), time.Now().Add(10*time.Second))
-	req, err := http.NewRequestWithContext(ctx, http.MethodGet, fmt.Sprintf("%s/load_balancer/%s", b.backendApiUrl, endpoint), nil)
+	req, _ := http.NewRequestWithContext(ctx, http.MethodGet, fmt.Sprintf("%s/load_balancer/%s", b.backendApiUrl, endpoint), nil)
```
Comment (adshmh) [inline cmd/apiserver/main.go]: Could we add a check on the error, instead of ignoring it?

File: api/server_test.go [hunk 3]
```diff
@@ -373,6 +381,108 @@ func TestHandleLoadBalancerRelays(t *testing.T) {
...
+
+			if resp.Header.Get("Content-Type") != "application/json" {
+				t.Errorf("Expected Content-Type: %s, got: %s", "application/json", resp.Header.Get("Content-Type"))
+			}
+
+			var r []LoadBalancerRelaysResponse
+			if err := json.Unmarshal(body, &r); err != nil {
+				t.Fatalf("Unexpected error unmarhsalling the response: %v", err)
+			}
+			if r[0].Count != tc.meterResponse[0].Count {
```
Comment (adshmh) [inline api/server_test.go]: Not a blocker, but this would look much better with a full comparison of the expected and actual arrays of relays.

**Review by adshmh — CHANGES_REQUESTED** [review]
Thank you for the PR! Just a few minor comments.

**Review by adshmh — APPROVED** [review]
LGTM. Thanks!

**Pull request merged**